offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/update/UpdateOfferViewModel.groovy

```diff
@@ -52,6 +41,7 @@ class UpdateOfferViewModel extends CreateOfferViewModel{
     }
 
     private void loadData(Offer offer) {
+        super.resetViewRequired.emit("reset")
```

Comment:
Why did you decide to go for an event emitter rather than directly calling the resetViewContent method?

Reply:
nice catch. Because i was not aware of that the method is accessible in the super class

Reply:
ah no

Reply:
because i wanted a nicer way to inform the view about the event, rather than a boolean property

Reply:
its not sufficient to reset the model, we also need to reset the navigation in the view, thats the second issue that is fixed

Reply:
I think this is too complicated. Why not call the method that this class already has access to? Since the view model extends the create offer view model it can simply call the reset method itself? Maybe I am missing something?

Reply:
it is not only the model, also the view needs to rewind and reset! How else would the view know? You can also use a property binding, but for me, thats as ugly as it can get to have an alternating trigger. The event emitter is explicit.